Dockerfile

```diff
@@ -71,13 +59,25 @@ ENV CXX $CXX
 RUN . $NVM_DIR/nvm.sh && nvm install $NODE_VERSION && nvm use $NODE_VERSION && \
     npm install -g node-gyp && node-gyp install
 
+# Copy sources
+COPY . .
```

Comment:
This Docker instruction copy from the local file system to the container file system. (https://docs.docker.com/engine/reference/builder/#copy)

in `+WORKDIR /usr/src/app` we tell docker that every command executed from now on will be relative to that folder, so `COPY . .` is equivalent to `COPY . /usr/src/app`